README.md

```diff
@@ -14,6 +14,9 @@ Whichever TTL is reached first will be used by the controller to delete the name
 The controller will will only consider pods matching the `!ci.openshift.io/ttl.ignore` selector for soft TTL calculations, allowing
 non-matching pods to keep running while considering the namespace to be inactive.
 
+Inactive namespaces are being marked for deletion from the last pod's transition time, plus the softTTL time duration.
+However, if the `ci.openshift.io/active` annotation exists and it's after the latest pod's transition time, the controller will use that value for the calculations instead.
```

Comment:
I agree that explaining the soft TTL again is a bit redundant. This is a good place to document the problem we are solving by introducing this annotation.